spec/models/page_spec.rb

```diff
@@ -1,49 +1,64 @@
 require "rails_helper"
 
 RSpec.describe Page, type: :model do
-  let(:response_data) do
-    {
-      id: 1,
-      question_text: "Question text",
-      answer_type: "date",
-    }.to_json
+  it "has a valid factory" do
+    page = build :page
+    expect(page).to be_valid
   end
 
-  let(:req_headers) do
-    {
-      "X-API-Token" => Settings.forms_api.auth_key,
-      "Accept" => "application/json",
-    }
-  end
+  describe "#answer_settings" do
+    it "returns an empty object for answer_settings when it's not present" do
+      page = build :page
+      expect(page).to have_attributes(answer_settings: {})
+    end
 
-  before do
-    ActiveResource::HttpMock.respond_to do |mock|
-      mock.get "/api/v1/forms/2/pages/1", req_headers, response_data, 200
+    it "returns an answer settings object for answer_settings when present" do
+      page = build :page, answer_settings: { only_one_option: "true" }
+      expect(page.answer_settings.attributes).to eq({ "only_one_option" => "true" })
     end
   end
 
-  it "returns the page for a form" do
-    expect(described_class.find(1, params: { form_id: 2 })).to have_attributes(id: 1, question_text: "Question text", answer_type: "date")
-  end
+  describe "repeatable?" do
+    it "when attribute does not exist" do
```

Comment:
I think this and the two `it` blocks below should be named to say something about what they expect - currently I think they're written to sound more like context names:

```suggestion
    it "returns false when attribute does not exist" do
```